go.mod

```diff
@@ -27,3 +27,5 @@ require (
 	k8s.io/apimachinery v0.21.3
 	k8s.io/client-go v0.21.3
 )
+
+replace github.com/theupdateframework/go-tuf => /home/asraa/git/go-tuf
```

Comment:
understand why you have this (local fork), could it point to a the online fork instead (I can play around with it then).

Reply:
Sure! Just FYI the go-tuf changes need to be pulled in